utils/config/config_test.go

```diff
@@ -161,6 +166,32 @@ func TestGetArtifactoriesFromConfig(t *testing.T) {
 	}
 }
 
+func TestGetJfrogDependenciesPath(t *testing.T) {
+	// Check default value of dependencies path, should be JFROG_CLI_HOME/dependencies
+	dependenciesPath, err := GetJfrogDependenciesPath()
+	if err != nil {
+		t.Error(err.Error())
+	}
+	jfrogHomeDir, err := GetJfrogHomeDir()
+	if err != nil {
+		t.Error(err.Error())
+	}
+	expectedDependenciesPath := filepath.Join(jfrogHomeDir, JfrogDependencies)
+	if strings.Compare(expectedDependenciesPath, dependenciesPath) != 0 {
```

Comment:
Can be probably shortened to:
*if expectedDependenciesPath != dependenciesPath {*
(two occurrences in this function).